src/Models/App.php

```diff
@@ -3,8 +3,89 @@
 namespace dnj\ErrorTracker\Laravel\Server\Models;
 
 use dnj\ErrorTracker\Contracts\IApp;
+use dnj\ErrorTracker\Database\Factories\AppFactory;
+use dnj\ErrorTracker\Laravel\Server\Kernel\DatabaseFilter\Traits\Filterable;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
 
+/**
+ * @property int        id
+ * @property string     title
+ * @property array|null extra
+ * @property int|null   owner
+ * @property string     owner_id_column
+ * @property \DateTime  created_at
+ * @property \DateTime  updated_at
+ */
 class App extends Model implements IApp
 {
-    // TODO
+    use Filterable;
+    use HasFactory;
+
+    protected $fillable = ['title'];
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
+    public function setTitle(string $title): void
+    {
+        $this->title = $title;
+    }
+
+    public function getExtra(): ?array
+    {
+        return $this->extra;
+    }
+
+    public function setExtra(?array $extra): void
+    {
+        $this->extra = json_encode($extra);
```

Comment:
Please [$casts](https://laravel.com/docs/9.x/eloquent-mutators#array-and-json-casting)